arches/app/templates/views/components/widgets/reference-select.htm

```diff
@@ -0,0 +1,55 @@
+{% extends "views/components/widgets/base.htm" %}
+{% load i18n %}
+
+{% block form %}
+<div class="row widget-wrapper" data-bind="class: nodeCssClasses">
+    <div class="form-group">
+        <span class="control-label widget-input-label" data-bind="text:label, class: nodeCssClasses"></span>
+        <!-- ko if: node -->
+        <i data-bind="css: {'ion-asterisk widget-label-required': node.isrequired}"></i>
+        <!-- /ko -->
+        <div class="col-xs-12 resource-instance-wrapper" data-bind="class: nodeCssClasses">
+            <select style="display:inline-block;"
+                data-bind="
+                    disable: false,
+                    select2Query: {
+                        select2Config: select2Config
+                    },
+                    attr: {'data-label': label}
+                ">
+        </div>
+    </div>
+</div>
+{% endblock form %}
+
+{% block config_form %}
+<div>
+    <div class="control-label">
+        <span data-bind="text: $root.translations.placeholder"></span>
+    </div>
+    <div class="col-xs-12 pad-no crud-widget-container">
+        <input
+            class="form-control input-md widget-input"
+            data-bind="
+                attr: {placeholder: $root.translations.placeholder, 'aria-label': $root.translations.placeholder},
+                value: placeholder,
+                valueUpdate: 'keyup'
+            "
+        >
+    </div>
+</div>
+{% endblock config_form %}
+
+{% block report %}
+<dt data-bind="text: label, class: nodeCssClasses"></dt>
+<!-- ko foreach: value -->
+    <dd data-bind="class: $parent.nodeCssClasses">
+        <a data-bind="attr:{'href': uri}, text: $parent.getPrefLabel(labels), style: {'color': 'steelblue'}"></a>
+    </dd>
+<!-- /ko -->
+<!-- ko ifnot: value() -->
+<dd data-bind="class: nodeCssClasses">
+    <span data-bind="text: $root.translations.none"></span>
+</dd>
+<!-- /ko -->
+{% endblock report %}
```

Comment:
What is the expected behavior when deleting the tile data without deleting the resources? My understanding is that the tile remains, and we just log out this error to arches.log:

```py
2024-02-05 22:32:43,918 arches.app.datatypes.base ERROR    Tile has no authoritative or provisional data
NoneType: None
```

Is that right?

Reply:
Do you mean deleting the a tile's data without deleting the tile, or all of a resources tiles without deleting the resource?

Reply:
Sorry -- deleting tile data. Basically using the widget to clear a saved value, then clicking save edit.

Reply:
Yeah - that's right then. The tile persists even if you used the widget to clear out all of the data on all of the nodes of the tile.

Reply:
Thanks, opened #10607 for it.